### PR TITLE
Make RH10 work with KA10

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
As a stepping stone towards preparing for running ITS on a KL10, I decided to try using the RH10 Massbus controller on a KA10.  Because MIT-MC was booted off an RP04 which is a Massbus device.

The experiment turned out well.  I have updated SALV, DSKDMP, NTSDDT, and ITS itself to support the KA10 + RH10 combination.

### CODE REVIEW

**Comment by larsbrinkhoff** [comment]
The plan is to rework and clean up this branch.  There will be a "pdp10-kl" target which despite its name is just another name for pdp10-ka.  Similarly there will be a KL ITS which is actually running on a KA10, but using RH10 and RP04 for disk storage.

**Comment by rcornwell** [comment]
Lars do not use pdp10-kl since this might be used for the KL10 version of the simulator.

**Comment by larsbrinkhoff** [comment]
The plan is that this is part preparation for KL10 ITS, and that pdp10-kl will indeed be the KL10 simulator when the time is right.

**Comment by rcornwell** [comment]
That is ok... however the KL10 used DF10C's not the modified DF10's that ITS used.

**Comment by larsbrinkhoff** [comment]
That's fine. I have left the KL10 and DF10-C code in place.

File: src/system/ddtdsk.32 [hunk 1]
```diff
@@ -250,9 +250,11 @@ wrtuf1:	skipe qact(a)
 
 ; Reset the disks
 
-reset:	coni pag,b		;turn off paging
```
[the remark this answers is not in the document]
Comment (larsbrinkhoff) [reply src/system/ddtdsk.32]: We don't.  It's now inside `kl.[` ... `]`instead, which means paging is *just* enabled for KL.

[the remark this answers is not in the document]
Comment (eswenson1) [reply src/system/ddtdsk.32]: Yeah. Thanks I figured that out when I viewed the whole file. Sorry.

**Pull request merged**